tutorials/secret-manager-generate-password/index.mdx

```diff
@@ -0,0 +1,108 @@
+---
+meta:
+  title: Generate password with Secret Manager
+  description: Learn how to generate password with Scaleway Secret Manager
+content:
+  h1: Generate password with Secret Manager
+  paragraph: Learn how to generate password with Scaleway Secret Manager
+tags: secret-management password-generation
+categories:
+  - secret-manager
+dates:
+  validation: 2023-05-24
+  posted: 2023-05-24
+---
+
+With the Secret Manager, when creating a version of the secret, you can ask to generate a password.
+
+## With the Scaleway CLI
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+- You have an account and have configured the [Scaleway CLI](https://www.scaleway.com/en/cli/)
+</Message>
+
+Here are the steps you need to follow to generate a password and store is securely in the Secret Manager:
+
+1. Create the secret by running the following command:
+    ```
+    scw secret secret create project-id=<your project ID> name=test-password
+    ```
+    Copy the secret ID that is shown in the output of the command.
+
+2. Generate the password by running the following command:
+    ```
+    scw secret version generate-password secret-id=<the created secret ID> length=12
+    ```
+
+    This will generate a password of length 12 and store it as a new version of the created secret.
+
+3. Get the generated password with the following command:
+    ```
+    scw secret version access secret-id=<the created secret ID> revision=latest
+    ```
+
+    This will print the base64 encoded password.
+
+## With the Scaleway go SDK
+
+<Macro id="iam-requirements" />
+
+With the following code, you can create a password and generate a password as a new version of that secret.
```

Comment:
```suggestion
The following code allows you to create a Scaleway account, a secret and to generate a password as a new version of that secret.
```

Reply:
This is an example code to create a secret with the SDK and then generate a password as a new version